shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -16,6 +16,22 @@
 class PointerDataPacket {};
 }
 
+/// Sometimes we have to use a custom mock to avoid retain cycles in ocmock.
+@interface FlutterEnginePartialMock : FlutterEngine
+@property(nonatomic, strong) FlutterBasicMessageChannel* lifecycleChannel;
+@property(nonatomic, weak) FlutterViewController* viewController;
+@property(nonatomic, assign) BOOL didCallNotifyLowMemory;
+@end
+
+@implementation FlutterEnginePartialMock
+@synthesize viewController;
+@synthesize lifecycleChannel;
+
+- (void)notifyLowMemory {
+  _didCallNotifyLowMemory = YES;
+}
+@end
```

Comment:
@jmagman This is what I had to do in order to get rid of the cycles.  I would hope that upgrading ocmock would also fix this, but it was hard to upgrade ocmock because it changed its dependencies, it's complicated to get GN working with it.